include/logger/common.hpp

```diff
@@ -22,5 +22,15 @@ struct ExtractedLogCredentials
   std::string function_name = "";
   int line_number = 0;
 };
+
+/**
+ * @brief  Data Structure to user specified log related details.
+ */
+struct LogSettings
+{
+ logger::LEVEL desired_level=logger::LEVEL::TRACE;
+ std::string logger_name="default logger";
+
```

Comment:
remove space

Reply:
along with this please check all over the places, where unnecessary line spaces are used.